docs/manual/layout/templates/twig/data.en.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Display template data'
+description: 'Display all template data.'
+aliases:
+    - /en/layout/templates/twig/template-data/
+weight: 60
+---
+
+
+To see what data is a available in a template (we call it the "context"), you can use the `dump()` function. If you only want to 
+know more about a specific part of the context, you can pass it as an argument:
+
+```twig
+{% extends "@Contao/content_element/text.html.twig" %}
+
+{% block content %}
+
+  {% set varA = 'My first <br> Text' %}
+  {% set varB = ['My second Text', 'My third Text'] %}
+
+  {{ dump(varA, varB) }}
+	
+  <ul>
+    {% for item in varB %}
+      <li>{{ item }}</li>
+			
+      {{ dump(loop) }}
+    {% endfor %}
+  </ul>
+
+  {% set myDebugBlock %}{{ text|raw }}{% endset %}
+
+  {{ dump(myDebugBlock) }}
+
+  {{ dump() }}
+
+{% endblock %}
```

Comment:
Isn't this way too complicated for this section? People want to understand how the `dump()` function works but we throw all sorts of unrelated things at them (blocks, set, extending a template, …).

IMHO the previous example had it all:
```suggestion
{{ dump() }}
{{ dump(varA) }}
{{ dump(varA, varB) }}
```